sdk/examples/devmode_rust/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "sawtooth-devmode-rust"
 version = "0.1.0"
 authors = ["Intel Corporation"]
+description = "Hyperledger Sawtooth DevMode consensus engine"
```

Comment:
Description should match other description in service file. "Sawtooth Devmode Rust Consensus Engine" seems good